docs/user-manual/en/jwt_security.md

```diff
@@ -2,13 +2,13 @@
 
 To leverage JWT security features, a pair of RSA keys (a public one and a private one) must be loaded in Kapua.
 
-## Use default Kapua keys
+## Use random generated keys
 
-A pair of RSA keys is already shipped within Kapua, and can be used **ONLY FOR DEVELOPMENT PURPOSES SINCE THEY ARE PUBLICLY AVAILABLE IN THE KAPUA REPOSITORY**. These keys will be automatically loaded when the system properties `authentication.session.jwt.private.key` and `authentication.session.jwt.public.key` are not set, or if the file entered in said properties cannot be read.
+By default Kapua will look for keys in the path specified by `authentication.session.jwt.private.key` and `authentication.session.jwt.public.key` system properties at startup (see below). If such properties are not set, or if Kapua can't find those files, it will automatically generate new random private and public keys.
```

Comment:
I would say that if those files are configured, but not existing then this is a major error! Silently falling back to generated keys should not be done in this case.